addons/io_scene_gltf2/__init__.py

```diff
@@ -431,10 +443,18 @@ def invoke(self, context, event):
         return ExportHelper.invoke(self, context, event)
 
     def save_settings(self, context):
-        # find all export_ props
+        # find all props to save
+        exceptional = [
+            # options that don't start with 'export_'
+            'use_selection',
+            'export_loose_edges',
+            'export_loose_points',
```

Comment:
I thought these were going to be consistent with the FBX exporter, like [the comment above](https://github.com/KhronosGroup/glTF-Blender-IO/pull/717#discussion_r430054075) said.

Also like the comment says, this is only for option names that don't start with `export_` so no need to put them here if they're named like this.